mm/alignment.py

```diff
@@ -413,3 +417,246 @@ def run(self):
                         total_score += float(e.find("./score/total").text)
 
                 f.write("%f\n" % (total_score / total_frames))
+
+
+class ComputeTSEJob(Job):
+    """
+    Compute TSE of some alignment compared to a reference
```

Comment:
I would underline that this is a word-level TSE and underline how this is computed in case of HMM topology with silence and blank-based models.
Maybe with the reference to our HMM vs CTC paper?

Reply:
Yes, thanks, I updated the docstring and made the description more detailed.